modules/ossm-rn-fixed-issues.adoc

```diff
@@ -41,3 +41,14 @@ To check HTTPS or TCP traffic by using an ingress gateway, you must have an exis
 * link:https://issues.jboss.org/browse/OSSM-93[OSSM-93] IstioConfigList can't filter by two or more names.
 +
 * link:https://issues.jboss.org/browse/OSSM-99[OSSM-99] Workloads generated from direct Pod without labels may crash Kiali.
+
+[id="ossm-rn-fixed-issues-maistra_{context}"]
+== Maistra fixed issues
+
+* link:https://issues.jboss.org/browse/MAISTRA-932[MAISTRA-932] Added the *requires* metadata to add dependency relationship between Jaeger operator and Elasticsearch operator. Ensures that when the Jaeger operator is installed, it automatically deploys the Elasticsearch operator if it is not available.
```

Comment:
Use backticks for "requires" instead of asterisks.

Throughout: s/operator/Operator/